src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj

```diff
@@ -110,23 +113,23 @@
     <PackageReference Include="Uno.Microsoft.Toolkit.Uwp.DeveloperTools" Version="6.1.0-build.205.g2e0e348372" />
     <PackageReference Include="Uno.UI" Version="3.11.6" />
     <PackageReference Include="Uno.UI.RemoteControl" Version="3.11.6" />
-    <PackageReference Include="Uno.Injectable" Version="1.34.0-dev.175" />
-    <PackageReference Include="Uno.CodeGen" Version="1.34.0-dev.175" />
+    <PackageReference Include="Uno.Injectable" Version="2.0.0-dev.10" />
```

Comment:
Do we need that in this same PR? It really feels like there's too much going on at the same time.

Reply:
Yes, it's needed because the build is failing without the identical version number

Reply:
we agree it shouldn't have been done in same PR but we left it for this one.